src/engine/ruleset/decoders/fortinet/client-endpoint.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: decoder/fortinet-client-endpoint/0
```

Comment:
In order to be clearer, the decoder name should match the file name. 
name: decoder/fortinet-client-endpoint/0 -> fortinet-client-endpoint.yml
The same for the rest of the files.